window/UserControlHost.vb

```diff
@@ -0,0 +1,93 @@
+﻿Imports System.Runtime.InteropServices
+Imports System.Windows.Forms
+Imports System.Drawing
+
+<ComVisible(True), Guid("0EB93108-D229-4F6F-82C5-0B96AFFBB9C5"), ProgId("MyVBAAddin.UserControlHost")>
+Public Class UserControlHost
+    Private Class SubClassingWindow
+        Inherits System.Windows.Forms.NativeWindow
+        Public Event CallBackProc(ByRef m As Message)
+
+        Public Sub New(ByVal handle As IntPtr)
+            MyBase.AssignHandle(handle)
+        End Sub
+
+        Protected Overrides Sub WndProc(ByRef m As Message)

```

Comment:
Please add proper method name. It should be readable.

Reply:
Can't change the name because it is a built-in function which takes the operating system message. Here i just override it.
You can see [this]( https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.wndproc?view=netcore-3.) here.